.gitignore

```diff
@@ -59,3 +59,6 @@ htmlcov
 
 # PyCharm
 .idea
+
+# Pytest
+*/cache/lastfailed
```

Comment:
I think this should just ignore the top-level `v` directory since nothing under it should be committed.

Reply:
You mean just putting `v/` there?

Reply:
Yes, that should be sufficient. I'm not sure why that's happening for you since I **think** it should be controlled by `XDG_CACHE_HOME` (why do I feel like we've had this conversation before???). But maybe this is another Windows-specific issue? In any case, I don't think it hurts anything to add this to `.gitignore`.

Reply:
But it seems very weird if one ignores `v` 😄